### PR TITLE
enhancement 1503 - caps lock warning

### PR DESCRIPTION
**Pull request by juanfran**

### CODE REVIEW

**Comment by bameda** [comment]
- the alert never disappears

**Comment by juanfran** [comment]
done

File: app/partials/includes/modules/invitation-register-form.jade [hunk 1]
```diff
@@ -1,7 +1,7 @@
 form.register-form
     p.form-header(translate="REGISTER_FORM.TITLE")
     fieldset
-        input(type="text", name="username", ng-model="dataRegister.username",
+        input(type="text", tg-capslock, name="username", ng-model="dataRegister.username", tg-capslock,
```
Comment (superalex) [inline app/partials/includes/modules/invitation-register-form.jade]: tg-capslock is duplicated and the linter is blaming about it :)

**Comment by superalex** [comment]
It looks great but i found some issues:

- Tests are failing...
- I don't see it working in the Forgot password
- If I write a capital letter and remove it the alert doesn't disappear

**Comment by juanfran** [comment]
This is "old-taiga" it doesn't have tests.

I've fixed the others, could you try again?

**Comment by superalex** [comment]
@bameda are you checking this one? (you are assigned), if not I can review it.

**Comment by superalex** [comment]
Great! :+1:

**Pull request merged**